frontend/src/routes/Applications/ApplicationDetails/ApplicationOverview/ApplicationOverview.test.tsx

```diff
@@ -230,7 +231,7 @@ const mockApplicationDataSubscription: ApplicationDataType = {
               'magchen-sibling-ns/magchen-sibling-subscription-1-local',
             'apps.open-cluster-management.io/reconcile-option': 'merge',
           },
-          creationTimestamp: '2022-03-01T21:30:02Z',
+          creationTimestamp: getShortDateTime('2022-03-01T21:30:02Z'),
```

Comment:
This change doesn't seem to be needed, and doesn't make sense to me. The code should call this function, not the test data.